app/views/start/index.html.slim

```diff
@@ -73,10 +73,5 @@
     hr
```

Comment:
Do we still need the `- if ChangeLogs.occurred.any?` as I assume we will always show this bit of text?

Reply:
That's a fair point - that was put in when we first built this logic and we didn't have any change logs, but we'll never now _not_ have any. I'll get rid of it.

Reply:
Oh, I'm wrong: a bunch of the tests "time travel" to points before we had any change logs, and then break when the start page is viewed.

Reply:
Oh ok! 
Do we then,
- leave a comment in for future devs?
or
- rewrite the tests?

Reply:
I'm inclined to leave it for now, on the grounds that we'll have to rewrite this code anyway when Change Logs become database objects, and when they are it'll be much more obvious that we shouldn't assume that they will exist.